### PR TITLE
Added Initiator Info and Table Header

### PR DESCRIPTION
**Pull request by TrezorHannes**

### CODE REVIEW

File: anchor_check.sh [hunk 1]
```diff
@@ -75,12 +72,13 @@ if [[ $1 == "--peer" ]]; then
...
 
     # Print SAT_VBYTE value
-    echo "$ANCHOR $SAT_VBYTE sat/vb "$'\t'" $ALIAS ($PUBKEY)"
+    echo "$INITIATOR $ANCHOR $SAT_VBYTE sat/vb "$'\t'" $ALIAS ($PUBKEY)"
 
 else
     # Check if --type option is provided
     if [[ $1 == "--type" ]]; then
-        if [[ $2 == "anchors" ]]; then
+        if
```
Comment (blckbx) [inline anchor_check.sh:80]: probably typo: line break after if?

**Review by blckbx — COMMENTED** [review]
utACK, one formatting nit

File: anchor_check.sh (hunk 1 above)
Comment (TrezorHannes) [reply anchor_check.sh]: Oh good catch, ty

**Review by TrezorHannes — COMMENTED** [review]
Fixed

**Review by blckbx — APPROVED** [review]
💥

**Pull request merged**